src/osgPlugins/dae/daeReader.cpp

```diff
@@ -300,6 +300,7 @@ bool daeReader::convert( std::istream& fin )
     // use a vector as buffer and read from stream
     std::vector<char> buffer(length);
```

Comment:
Initialize to `length + 1` to avoid reallocation?
Then there is no need to append '\0' either.

Reply:
I've sent #1024 with this and other fixes